src/index.html

```diff
@@ -29,4 +29,4 @@ <h3 id="clear">Clear All Completed</h3>
             </div>
         </div>
     </div>
-</body>
\ No newline at end of file
+</body>
```

Comment:
- [ ] There is no file import with methods related to status updates(completed: true/false). Please add a new javascript file and put status update methods in it. Then import it as a JS module.

Reply:
Hi @wuletawwonte I do not understand, kindly come again

Reply:
Hi @altontonn 
The first requirement is not implemented. I think you are supposed to create a new javascript file and write your status update methods(Checkbox status change handler methods) in it. Then import it as a module in the index.js file.